src/components/transport_manager/src/usb/libusb/usb_connection.cc

```diff
@@ -206,6 +206,14 @@ bool UsbConnection::PostOutTransfer() {
 
 void UsbConnection::OnOutTransfer(libusb_transfer* transfer) {
   LOG4CXX_TRACE(logger_, "enter with  Libusb_transfer*: " << transfer);
+
+  if (transfer->status == LIBUSB_TRANSFER_CANCELLED) {
+    LOG4CXX_DEBUG(logger_,"Free already canceled transfer.");
+    libusb_free_transfer(transfer);
+    LOG4CXX_TRACE(logger_,"exit");
```

Comment:
@Ypostolov please add `LOG4CXX_AUTO_TRACE` at line 208 and remove this one at 213

Reply:
fixed in 8977c019208ef0b106eb7361f53f64d825ad4682